### PR TITLE
Update auto-generated documentation

### PR DESCRIPTION
**Pull request by snoopycrimecop**
Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by build [OMERO-DEV-latest-docs-autogen#1149](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1149/).

----
--no-rebase

### CODE REVIEW

**Comment by hflynn** [comment]
This will be green once https://github.com/openmicroscopy/openmicroscopy/pull/5630 is merged and the job is re-run

**Comment by joshmoore** [comment]
Hmmm.... merged and ran https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1150 but the update here still has "extensions".

**Comment by hflynn** [comment]
Ah! I think the code build needs to run again before the changes can be ported across. Should be green tomorrow morning.

**Comment by joshmoore** [comment]
Green! Holding off on merging in case we notice anything else but if not, we're ready to go.

**Comment by hflynn** [comment]
Yeah, safest not to merge until all the code PRs are merged and the code build re-run just incase there's something else to come across in the configs or whatever.

File: omero/users/history.rst [hunk 1]
```diff
@@ -5,6 +5,55 @@
...
+-  extended Python and Java examples to include Map Annotations and histograms
+-  added methods for updating OMERO.tables
+-  Java Gateway fixes for sessions and rendering
+-  fixed retrieval of Plate thumbnail URLs
+-  improved 'Editing OMERO.web' documentation
+-  improved Slice documentation for API deprecations
+-  added instructions to :doc:`/developers/cli/extending` on how to
+   create CLI plugins that are ``pip`` installable
+-  substantial effort to make third-party repositories easily testable;
+   see `omero-test-infra <https://github.com/openmicroscopy/omero-test-infra>_`
```
Comment (hflynn) [inline omero/users/history.rst]: this is malformed and doesn't work in the built docs - see https://github.com/openmicroscopy/openmicroscopy/pull/5632 for the fix

**Comment by hflynn** [comment]
Merge docs look good - all green and page renders as expected now

**Pull request merged**